### DISCUSSION TITLE
Backebd web gui

### DISCUSSION THREAD

**Discussion by jaas666** [simple-login/app#2322] [Self-hosting question]
I'm deploying this at home and my goal is to have my friends and family use it.

Is there some kind of backend interface to manage their accounts or it's all done trough the database?

**Comment by nguyenkims** [comment]
You can add /admin to the URL to have access to the admin panel. Before that you need to set your account as admin by setting the admin column in the database to true.

**Reply by jaas666** [reply]
That worked perfectly, just one note. The correct column in the `users` table is named `is_admin`.

Thanks a lot. 👍
